DatadogRUM/Sources/RUM+Internal.swift

```diff
@@ -19,4 +64,56 @@ extension InternalExtension where ExtendedType == RUM {
     public static func isEnabled(in core: DatadogCoreProtocol = CoreRegistry.default) -> Bool {
         return core.get(feature: RUMFeature.self) != nil
     }
+
+    /// Enable URL session tracking after RUM has already been enabled. This method
+    /// is only needed if the configuration of URL session tracking is not known at initialization time,
+    /// or in the case of cross platform frameworks that do not initalize native URL session tracking.
+    ///
+    /// - Parameters:
+    ///    - in: the configuration for URL session tracking
+    public static func enableURLSessionTracking(
+        with configuration: RUM.Configuration.LateURLSessionTracking,
+        in core: DatadogCoreProtocol = CoreRegistry.default) throws {
```

Comment:
question/ Have we considered changing this to following?
```swift
    public static func enableURLSessionTracking(
        with urlSessionTrackingConfiguration: RUM.Configuration.URLSessionTracking,
        rumConfiguration: RUM.Configuration,
        in core: DatadogCoreProtocol = CoreRegistry.default) throws {
```
Effectively, this way we won't need to introduce new `LateURLSessionTracking` nor duplicate any properties. The trade-off will be on the caller-side (in Flutter), where we'll need to supply this method both configurations:
- `RUM.Configuration.URLSessionTracking`
- `RUM.Configuration`

For the sake of streamlining RUM init, the cost seems worth it. WDYT?

Reply:
I'm concerned with requiring `RUM.Configuration` because of the number of parameters available that just aren't used, especially the non-optional parameters that wouldn't actually have an affect.

Remember "on the Flutter side" in this case is a client who has to make sense of the API, and requiring the `RUM.Configuration` when only a few parameters out of it are going to be used worries me. I'm open to discuss it though.

Reply:
I see 👍. One more idea that comes to my mind is attaching `RUM.Configuration` to `RUMFeature`, so we can retrieve it "after" by ourselves without requiring the client to pass it:
```swift
public static func enableURLSessionTracking(
   with urlSessionTrackingConfiguration: RUM.Configuration.URLSessionTracking,
   in core: DatadogCoreProtocol = CoreRegistry.default
) throws {
   guard let rum = core.get(feature: RUMFeature.self) else {
      throw ProgrammerError(
         description: "RUM must be initialized before calling `RUM.enableUrlSessionTracking`."
      )
   }
   let rumConfiguration = rum.configuration // <-- ⭐️ new thing
   // ...
```

This isn't a change request. Using separate `LateURLSessionTracking` type makes sense as well 👍.